tests/vereya/test_consistency.py

```diff
@@ -98,7 +61,7 @@ def test_drop_and_see(self):
         grid = mc.getNearGrid()
         oak_log_world_block_name = ""
         for grid_item in grid:
-            if "oak" in grid_item:
+            if "oak_log" in grid_item:
                 oak_log_world_block_name = grid_item
                 break
         for blockdrop in blockdrops:
```

Comment:
I would add additional check that during iteration "oak_log" was found in blockdrop

Reply:
Done